torch/_dynamo/testing.py

```diff
@@ -246,24 +245,33 @@ def __call__(
         return gm.forward
 
 
-# Equivalent to backend="aot_eager", but also records graphs that
-# we can assert on
-class AOTEagerAndRecordGraphs:
+class AotEagerAndRecordGraphs:
```

Comment:
Does the unittest above need to change to use this? I still see `backend = torch._dynamo.testing.AOTEagerAndRecordGraphs()` above.

Reply:
#137677 added this backend. The test above uses this backend. So I updated the test